README.md

```diff
@@ -63,11 +63,10 @@ If you haven't already setup K's OCaml dependencies more recently than February
 Finally, you can install repository specific dependencies and build the semantics:
 
 ```sh
-make deps
-make build
+./build
 ```
 
-To only build specific backends, you can do `make build-java`, `make build-ocaml`, or `make build-haskell`.
+To only build specific backends, you can do `./build wasm-java`, `./build wasm-ocaml`, or `./build wasm-haskell`.
```

Comment:
How about the tests? Should they also be built using `ninja`? I assume we wouldn't want to have *both* a Makefile and the ninja system.

Reply:
We were planning on having both systems for a little while, to make sure that we are not losing any features. I think in a few weeks, I would move the `Makefile` to the `media` directory and have it only for building the media targets (which currently KNinja does not cover).

The ninja build system is capturing dependencies more accurately, so running `./build test-exec-ocaml` will always check (1) ocaml deps installed, (2) ocaml version of semantics built, and then (3) run the tests.